quickstarts/library-process/README.md

```diff
@@ -0,0 +1,192 @@
+## Red Hat Business Automation Kie Server Quickstart
+
+This quickstart is intend to be used with the [RHBA Kie Server](https://github.com/jboss-container-images/rhba-7-openshift-image/tree/rhba70-dev/kieserver) image.
+
+## How to use it?
+
+To deploy the Library Process demo you can use the [rhba70-kieserver-s2i](https://github.com/jboss-container-images/rhba-7-openshift-image/blob/rhba70-dev/templates/rhba70-kieserver-s2i.yaml)
```

Comment:
shouldn't be rhba70-kieserver-basic-s2i.yaml?
I guess that the next lines of this readme also needs to be update to reflect the template name.

Reply:
The readme has a lot of work that needs to be done to it. I am issuing this PR to get something that can be accessible for others to start building with, but will go back and work on docs in another PR. I'll fix this still, though.